admin/sql/CreateTables.sql

```diff
@@ -576,6 +576,20 @@ CREATE TABLE edit_note
     post_time            TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
+CREATE TYPE edit_note_status AS ENUM ('deleted', 'edited');
```

Comment:
We actually have admin/sql/CreateTypes.sql now which this should probably be moved to